services/media/workflows/src/Stations/Movies/MovieLicensingDetails/MovieLicensingDetails.tsx

```diff
@@ -32,15 +32,21 @@ type FormData = MutationUpdateMoviesLicenseArgs['input']['patch'] & {
 };
 
 const licenseSchema = Yup.object().shape<ObjectSchemaDefinition<FormData>>({
-  licenseStart: Yup.date(),
-  licenseEnd: Yup.date().when('licenseStart', {
-    is: (start) => start != null,
-    then: (end) =>
-      end.min(
-        Yup.ref('licenseStart'),
-        'License end date cannot be before start date',
-      ),
-  }),
+  licenseStart: Yup.date().typeError('required'),
+  licenseEnd: Yup.date()
+    .typeError('required')
+    .test('checkEndDate', '', function (value) {
+      const { path, parent, createError } = this;
+      if (value) {
+        if (parent.licenseStart.getTime() >= value.getTime()) {
+          return createError({
+            path,
+            message: 'License end date cannot be before start date',
+          });
+        }
+      }
+      return true;
+    }),
```

Comment:
This can be simplified as, 

```suggestion
    .test(
      'checkEndDate',
      'License end date cannot be before start date',
      function (value) {
        const { parent } = this;
        if (value) {
          return parent.licenseStart.getTime() < value.getTime();
        }
        return true;
      },
    ),
```